ingest/ingest_pipeline.py

```diff
@@ -92,7 +92,7 @@ class IngestPipeline(object):
     JSON_CONVENTION = (
         '../schema/alexandria_convention/alexandria_convention_schema.json'
     )
-    EXTERNAL_JSON_CONVENTION = 'gs://broad-singlecellportal-public/schema/alexandria_convention/2.0.0/alexandria_convention_schema.json'
+    EXTERNAL_JSON_CONVENTION = 'gs://broad-singlecellportal-public/schema/alexandria_convention/2.0.1/alexandria_convention_schema.json'
```

Comment:
We're keeping this until we update `manage_study.py` to use the SCP REST API, correct?

Reply:
Yes, I checked with Eno and we agreed that it wouldn't make sense for me to remove `EXTERNAL_JSON_CONVENTION` until `manage_study.py` was using the API endpoint to obtain the metadata convention.